wcfsetup/install/files/acp/update_com.woltlab.wcf_5.2.10_orphanedComments.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace wcf\acp;
+
+use wcf\data\object\type\ObjectTypeCache;
+use wcf\system\comment\CommentHandler;
+use wcf\system\WCF;
+
+// Earlier versions did remove moderation queues without taking care of
+// comments associated with them.
+$commentObjectTypeID = ObjectTypeCache::getInstance()->getObjectTypeIDByName(
+	"com.woltlab.wcf.comment.commentableContent",
+	"com.woltlab.wcf.moderation.queue"
+);
+
+$sql = "SELECT  objectID
+        FROM    wcf" . WCF_N . "_comment
+        WHERE   objectTypeID = ?
+        AND     objectID NOT IN (
+                        SELECT  queueID
+                        FROM    wcf" . WCF_N . "_moderation_queue
+                )";
+$statement = WCF::getDB()->prepareStatement($sql);
+$statement->execute([$commentObjectTypeID]);
+$objectIDs = $statement->fetchList('objectID');
```

Comment:
```suggestion
$objectIDs = $statement->fetchAll(\PDO::FETCH_COLUMN);
```
I wasn't even aware that `fetchList()` existed.

Reply:
We should consider deprecating it then, I wasn't aware of `fetchAll()` for single column results.